test/e2e_test.go

```diff
@@ -535,6 +536,9 @@ func consumeCPU(client clientset.Interface, podName string) error {
 					Requests: map[corev1.ResourceName]resource.Quantity{
 						corev1.ResourceCPU: mustQuantity("100m"),
 					},
+					Limits: map[corev1.ResourceName]resource.Quantity{
```

Comment:
Can you leave a TODO to investigate and remove limit?

Reply:
done